src/Designer/frontend/packages/schema-editor/src/components/SchemaInspector/SchemaInspector.test.tsx

```diff
@@ -98,7 +98,7 @@ describe('SchemaInspector', () => {
 
     updatedModel = getSavedModel(saveDataModel, 7);
     updatedNode = updatedModel.getNodeBySchemaPointer(schemaPointer) as FieldNode;
-    expect(updatedNode.restrictions.minLength).toEqual(parseInt(minLength));
```

Comment:
This must have been a mistake done when writing this test. It didn't fail because the `StringRestrictions` component updated the object directly instead of creating a new one with updated values. Now that the updates are pure, and since the components are controlled, the object with which the `save` function is called does not differ from the original one with other things than the ones implied by the last action.

Ideally, this test should be split in two, but that would be out of scope of this pull request.